src/tokenizer.test.js

```diff
@@ -33,18 +29,18 @@ tokensShouldBe('SUM(A1:B2)', [
   new Token(TokenTypes.COLON, ':'),
   new Token(TokenTypes.NAME, 'B2'),
   new Token(TokenTypes.RPAREN, ')'),
-  new Token(TokenTypes.EOF),
+  new Token(TokenTypes.EOF)
```

Comment:
Looks like a bunch of these changes are unrelated. I think your editor removed the trailing commas here. I want to leave them to follow the [airbnb styleguide](https://github.com/airbnb/javascript).